examples/Example/src/swipeable/GmailStyleSwipeableRow.tsx

```diff
@@ -1,56 +1,56 @@
-import React, { Component } from 'react';
-import { Animated, StyleSheet, Text, View, I18nManager } from 'react-native';
+import React, {Component} from 'react';
+import {Animated, StyleSheet, I18nManager, View} from 'react-native';
 
-import { RectButton } from 'react-native-gesture-handler';
+import {RectButton} from 'react-native-gesture-handler';
 
 import Swipeable from 'react-native-gesture-handler/Swipeable';
-import Icon from 'react-native-vector-icons/MaterialIcons';
 
-const AnimatedIcon = Animated.createAnimatedComponent(Icon);
+const AnimatedView = Animated.createAnimatedComponent(View);
 
 export default class GmailStyleSwipeableRow extends Component {
-  renderLeftActions = (progress, dragX) => {
+  renderLeftActions = (
+    progress: Animated.AnimatedInterpolation,
+    dragX: Animated.AnimatedInterpolation
+  ) => {
     const scale = dragX.interpolate({
       inputRange: [0, 80],
       outputRange: [0, 1],
       extrapolate: 'clamp',
     });
     return (
       <RectButton style={styles.leftAction} onPress={this.close}>
-        <AnimatedIcon
-          name="archive"
-          size={30}
-          color="#fff"
-          style={[styles.actionIcon, { transform: [{ scale }] }]}
-        />
+        {/* Change it to some icons */}
+        <AnimatedView style={[styles.actionIcon, {transform: [{scale}]}]} />
       </RectButton>
     );
   };
-  renderRightActions = (progress, dragX) => {
+  renderRightActions = (
+    progress: Animated.AnimatedInterpolation,
+    dragX: Animated.AnimatedInterpolation
+  ) => {
     const scale = dragX.interpolate({
       inputRange: [-80, 0],
       outputRange: [1, 0],
       extrapolate: 'clamp',
     });
     return (
       <RectButton style={styles.rightAction} onPress={this.close}>
-        <AnimatedIcon
-          name="delete-forever"
-          size={30}
-          color="#fff"
-          style={[styles.actionIcon, { transform: [{ scale }] }]}
-        />
+        {/* Change it to some icons */}
```

Comment:
📢 🚨 

(just giving this comment attention it needs.)

Reply:
You can see the desired behavior with this, but would be nice for sure.

Reply:
We agreed this isn't a top priority and it works so closing.